PR for the contribution for bpython

```diff
@@ -0,0 +1,140 @@
+diff --git a/README.rst b/README.rst
+index ad90007..380ec8e 100644
+--- a/README.rst
++++ b/README.rst
+@@ -45,6 +45,10 @@ You can find more about bpython - including `full documentation`_ - at our
+ ==========================
+ Installation & Basic Usage
+ ==========================
++
++Installation using Pip
++----------------------
++
+ If you have `pip`_ installed, you can simply run:
+ 
+ .. code-block:: bash
+@@ -55,6 +59,58 @@ Start bpython by typing ``bpython`` in your terminal. You can exit bpython by
+ using the ``exit()`` command or by pressing control-D like regular interactive
+ Python.
+ 
++Installation via OS Package Manager
++-----------------------------------
++
++The majority of operating system of desktop computers comes with a package
++manager system, if you are any user of them, you can install ``bpython``
++using the package manager.
++
++Ubuntu/Debian
++~~~~~~~~~~~~~
++Ubuntu/Debian family Linux users and install bpython using the apt package manager, using the
++command:
++
++.. code-block:: bash
++
++    $ sudo apt-get install bpython
```

Comment:
Should be: `apt install bpython`. I think the whole section should be consistent with the use of `sudo`.